src/test/regress/expected/multi_foreign_key.out

```diff
@@ -882,8 +870,7 @@ SELECT create_reference_table('reference_table');
 (1 row)
 
 ALTER TABLE reference_table ADD CONSTRAINT fk FOREIGN KEY(referencing_column) REFERENCES referenced_local_table(id);
-ERROR:  cannot create foreign key constraint
-DETAIL:  Foreign key constraints are not allowed from or to reference tables.
+ERROR:  relation referenced_local_table is not distributed
```

Comment:
Since we check foreign table constraints after we create them, planning phase catch the not distributed table before constraint checking catch the reference table. I can get the change back by adding a check of reference tables before planning phase, yet it will break the logic of checking constraints of ALTER TABLE and CREATE TABLE statements using the same code part.